pygerm/client/__init__.py

```diff
@@ -11,7 +11,7 @@ def payload2event(data):
     The layout is
 
        "0" [[4 bit chip addr] [5 bit channel addr]] [10 bit TD] [12 bit PD]
-       "1000" [28 bit time stamp]
```

Comment:
The 28 bit is the updated version, the 31 bit table was from an old version of the docs from the detector group.

Reply:
thanks!